gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FileExporter.java

```diff
@@ -136,8 +137,8 @@ private Object getVoltageMonitoringResult(VoltageMonitoringResult result) {
     }
 
     public String zipTargetNameChangeExtension(String targetName, String extension) {
-        if (StringUtils.isNotBlank(targetName) && targetName.toLowerCase().contains(".zip")) {
-            return targetName.replace(".ZIP", extension).replace(".zip", extension);
+        if (StringUtils.isNotBlank(targetName) && targetName.toLowerCase().contains(ZIP)) {
+            return targetName.replace(".ZIP", extension).replace(ZIP, extension);
```

Comment:
Maybe use `StringUtils.replaceIgnoreCase` to avoid replacing twice, once with the extension in capital letters inlined in the method call and once with extension in small letters extracted in a constant which is written in capital letters...

Reply:
ok